README.md

```diff
@@ -1,41 +1,43 @@
 ![masonry](https://i.imgur.com/AVeTw1M.png)
 
 # masonry
+
 A simple masonry library written in TypeScript.
 
 The masonry library we need, but don't deserve. Uses pure JavaScript and should be compatible with most browsers.
 
 ## Installation
 
 npm
+
 ```bash
-npm i @fristys/masonry
+npm i @gyurielf/masonry
 ```
 
 yarn
+
 ```bash
-yarn add @fristys/masonry
+yarn add @gyurielf/masonry
 ```
 
 ## Usage
 
 ```typescript
-import { Masonry } from '@fristys/masonry';
+import { Masonry } from '@gyurielf/masonry';
```

Comment:
@gyurielf You do know that this project is licensed, right? You've seen the license file?